README.md

```diff
@@ -0,0 +1,32 @@
+# feast-custom-stores-demo
+
+This is an example repo which implements a custom online store for Feast, and then uses it in a feature repo.
+
+## Installation
+
+This installation guide assumes you have mysql.server running locally already. 
+If not, please follow [this guide](https://flaviocopes.com/mysql-how-to-install/).
+
+This guide assumes running all the commands from the root of this repo, once it's cloned locally.
+
+- Create a virtualenv: 
+```
+virutalenv -p python3.7 env
+. env/bin/activate
+```
+- Install the `feast-custom-stores-demo` package
+```bash
+pip install -u pip
+pip install -e .
+```
+- Apply the feature repo:
+```bash
+cd feature_repo && feast apply
+```
+
+In the happy path, this should produce the following output:
+```bash
+Registered entity driver_id
```

Comment:
How does this output help to validate their connector though? Even a broken connector will work for registration purposes right? Shouldn't we read and write using the connector?

Reply:
We should yes - v0 of the readme